it/services/AddressLookupServiceIntegrationSpec.scala

```diff
@@ -1,18 +1,25 @@
 package services
 
 import controllers.testSupport.{ITCoreTestData, Specifications, TestConfiguration}
-import models.AlfResponse
+import models.alf.AlfResponse
+import models.alf.init.{JourneyConfig, JourneyOptions}
 import models.core.ErrorModel
 import org.scalatest.TryValues
 import play.api.http.Status
+import play.api.i18n.{Lang, MessagesApi}
+import play.api.libs.json.Json
+import play.api.test.{DefaultAwaitTimeout, FakeRequest, FutureAwaits}
 import uk.gov.hmrc.http.HeaderCarrier
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class AddressLookupServiceIntegrationSpec extends Specifications with TestConfiguration with  ITCoreTestData with TryValues  {
+class AddressLookupServiceIntegrationSpec extends Specifications with TestConfiguration with  ITCoreTestData with TryValues with FutureAwaits with DefaultAwaitTimeout {
   implicit val hc = HeaderCarrier()
+  val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
+  implicit lazy val messages = messagesApi.preferred(Seq(Lang("en")))
+
+  "getAddress" should {
 
-  "AddressLookupService" should {
     "should find the address in alf from the id given from alf" in {
```

Comment:
Minor cosmetic issue but these will show as two shoulds in the tests.
![image](https://user-images.githubusercontent.com/5639505/236434695-285bb796-348b-4b27-95e8-66463e3408f9.png)